### PR TITLE
textarea upgrade

### PR DESCRIPTION
**Pull request by matikrk**
* add nested

was, but moved for latter
* add autoHeightReact, done by react-textarea-autosize package

### CODE REVIEW

**Comment by matikrk** [comment]
At this PR only nested

File: src/components/form-elements/_textarea.scss [hunk 1]
```diff
@@ -85,6 +85,15 @@ $includeHtml: false !default;
       line-height: rhythm(1);
       height: auto;
     }
+
+    &--nested {
+      display: flex;
+      border: none;
+      border-radius: 0;
```
Comment (kdzwinel) [inline src/components/form-elements/_textarea.scss]: `--simple` already does `border: none + border-radius: 0`

File: src/components/form-elements/_textarea.scss [hunk 2]
```diff
@@ -85,6 +85,15 @@ $includeHtml: false !default;
...
+      padding: rhythm(0.104) 0;
+      height: rhythm(1);
+      width: 100%;
```
Comment (kdzwinel) [inline src/components/form-elements/_textarea.scss]: `--full-width` alrady does `width:100%`

**Comment by kdzwinel** [comment]
How does it look? Please add a test.

**Pull request closed without merging**